.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # Default owner(s) of all files in this repository
-* @cmsirbu @snaselj @whitej6
+* @nautobot/maintain-nautobot-apps
```

Comment:
This team needs to be given write access to this repo. I don't have admin access to this repo so I can't change it